test/truefi2/TrueFiCreditOracle.test.ts

```diff
@@ -41,24 +41,37 @@ describe('TrueFiCreditOracle', () => {
     })
   })
 
-  describe('set and get credit scores', () => {
+  describe('setScore', () => {
     const firstScore = 100
     const secondScore = 200
 
     beforeEach(async () => {
       await oracle.connect(manager).setScore(firstAccount.address, firstScore)
     })
 
-    it('only manager can set scores', async () => {
+    it('reverts if caller is not the manager', async () => {
       await expect(oracle.connect(owner).setScore(firstAccount.address, firstScore))
         .to.be.revertedWith('TrueFiCreditOracle: Caller is not the manager')
     })
 
-    it('score is set correctly for account', async () => {
+    it('sets correct score for the account', async () => {
```

Comment:
What was wrong with this and previous test names?

Reply:
My bad, I'm so used to this naming style I changed it subconsciously. I can change it back if you like.